### ISSUE TITLE
hyperechogenic liver

### ISSUE THREAD

**Issue by gawlinski** [obophenotype/human-phenotype-ontology#3622]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
hyperechogenic liver

**Comment by pnrobinson** [comment]
This term is synonymous. 
https://hpo.jax.org/app/browse/term/HP:0031141
ToDo: Add "Hyperechogenic liver" as an exact synonym.
